signing_clients/apps.py

```diff
@@ -26,6 +32,11 @@
 continuation_re = re.compile(r"""^ (.*)""", re.I)
 directory_re = re.compile(r"[\\/]$")
 
+RESIGN_IGNORE = ("META-INF/manifest.mf",
```

Comment:
I just knew this would be an issue.  RE_SIGN_IGNORE was even more confusing, sadly.